BasicTests/crypto.json

```diff
@@ -1,8 +1,16 @@
-[
-    {
+{
+    "aes1": {
```

Comment:
Good to see these tests :)  AES-CTR should probably include an IV in order to confirm the functionality of an IV is available. We omit the IV for streams but I believe it maybe used for ECIES.